README.md

```diff
@@ -36,6 +36,27 @@ Go to http://localhost:3000
 $ rails spec
 ```
 
+
+## Using Docker
+
+> NOTE: You'll need to have docker and docker-compose installed
+
+Build the points docker image
+```
+$ docker-compose build
+```
+
+Setup database and run migrations
+```
+$ docker-compose run web bundle exec rake db:setup
+$ docker-compose run web bundle exec rake db:migrate
+```
+
+Run the app
+```
+$ docker-compose up
+```
+
```

Comment:
I think we should also have instructions to run the app in the next rails version.